app.js

```diff
@@ -14,62 +17,82 @@ const catchphraseButton = document.getElementById('catchphrase-button');
 // set state for how many times the user changes the head, middle, and bottom
 // set state for all of the character's catchphrases
 
-headDropdown.addEventListener('change', () => {
-    // get the value of the head dropdown
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
+let catchPhrases = [''];
```

Comment:
Totally understand adding the string to your array here to illustrate the full datatype of your state. That being said, if you watch the site when you add your first catchphrase, there is an empty `<li />` that gets added first and this is because you have it starting out with an empty string in your state array.